JM/CF/Scripts/3_Game/CommunityFramework/Spline/CF_Spline.c

```diff
@@ -0,0 +1,264 @@
+class CF_Spline : Managed /*CF_ObservableCollection*/
```

Comment:
Remove comment if no longer needed?

Reply:
Currently not needed yes.

Will be in the future once `CF_ObservableCollection` is implemented.

Reply:
Not sure if things like this should BE a `CF_ObservableCollection`, maybe HAVE one for the points list?